contracts/pm/mixins/MixinReserve.sol

```diff
@@ -80,27 +74,8 @@ contract MixinReserve is MContractRegistry, MReserve {
      * @param _amount Amount of funds to add to reserve
      */
     function addReserve(address _reserveHolder, uint256 _amount) internal {
-        ReserveManager storage manager = reserveManagers[_reserveHolder];
-        Reserve storage reserve = manager.reserve;
-
-        reserve.fundsAdded = remainingReserve(_reserveHolder).add(_amount);
-
-        // If reserve is thawed then clear unneeded contract storage
-        // for the reserve and update the reserve while ensuring that
-        // any additional funds are added to the remaining funds in the reserve
-        if (reserveState(_reserveHolder) == ReserveState.Thawed) {
-            // We clear these individual fields of the reserve instead
-            // of using `clearReserve()` because we can directly update
-            // fundsAdded with its new value instead of clearing the old
-            // value and then setting its new value
-            reserve.fundsClaimed = 0;
-            reserve.freezeRound = 0;
-            reserve.recipientsInFreezeRound = 0;
-            // Increment reserveNonce so the manager can point to a new
-            // mapping for tracking funds claimed from a reserve by
-            // different claimants
-            manager.reserveNonce = manager.reserveNonce.add(1);
-        }
+        Reserve storage reserve = reserves[_reserveHolder];
```

Comment:
we could access the field directly since we're no longer accessing multiple fields on the `Reserve` struct within this function. I guess it saves us some stack operations of popping the storage address of the struct onto the stack.

`reserves[_reserveHolder].funds = reserves[_reserveHolder].funds.add(_amount);`

Reply:
Fixed in 1723500